android/app/build.gradle

```diff
@@ -256,9 +256,9 @@ android {
 dependencies {
     implementation("com.facebook.react:react-android")
     implementation 'com.facebook.soloader:soloader:0.10.1+'
-    implementation("io.mosip:pixelpass:0.2.0-SNAPSHOT")
-    implementation("io.mosip:secure-keystore:0.2.0-SNAPSHOT")
-    implementation("io.mosip:tuvali:0.5.0-SNAPSHOT")
+    implementation("io.mosip:pixelpass:0.2.0")
+    implementation("io.mosip:secure-keystore:0.5.0")
+    implementation("io.mosip:tuvali:0.5.0")
```

Comment:
Let's use patch released version for these dependencies
pixelpass - 0.2.1, 
secure keystore - 0.2.1
tuvali - 0.5.1